transportable-udfs-examples/transportable-udfs-example-udfs/build.gradle

```diff
@@ -17,7 +17,8 @@ dependencies {
 plugins.withId('com.github.hierynomus.license') {
   licenseHive.enabled = false
   licenseTrino.enabled = false
-  licenseSpark.enabled = false
+  tasks.named('licenseSpark_2.11').get().enabled = false
```

Comment:
why will we use different pattern? can we not setup the script like licenseSpark_2_11.enabled and licenseSpark_2_12.enabled ?

Reply:
unfortunately not. The license tasks are named automatically based on the module name. For example: "transportable-udfs-example-udfs:licenseSpark_2.11". Its not possible to get a dot-separated name inside of the plugins.withId(){} block.

Reply:
I've changed the pattern for all license tasks.

Looking at the comment it doesn't add any value to the user. Lemme know if you think otherwise.
```
plugins.withId('com.github.hierynomus.license') {
  // task names can have a dot, they cannot be mentioned directly without tasks.getByName 
  tasks.getByName('licenseTrino').enabled = false
  tasks.getByName('licenseHive').enabled = false
  tasks.getByName('licenseSpark_2.11').enabled = false
  tasks.getByName('licenseSpark_2.12').enabled = false
}
``` 
I can add the comment.